src/parsec_service/install_parsec_linux.md

```diff
@@ -24,23 +25,46 @@ git clone https://github.com/parallaxsecond/parsec.git
 cargo install --features $DESIRED_FEATURES --path parsec
 ```
 
-Copy and adapt the [configuration](configuration.md) you want to use.
+Copy and adapt the [configuration](configuration.md) you want to use. For a secure deployment, make
+sure to activate the `log_error_details` option and to use a `trace` log level.
 
 ```
 cp parsec/config.toml config.toml
 ```
 
+Create the Parsec socket directory.
+
+```
+mkdir /tmp/parsec
+```
+
+In a deployment **without an Identity Provider**, create the `parsec-clients` group and set the
```

Comment:
I think we need to set a bit of context here around identity provider, which may be an unfamiliar term and could confuse the reader. In fact, since we haven't integrated any identity providers yet, I'm wondering whether we should just assume the absence of it for now, and explain that the socket needs to be protected at deployment time because there is currently no runtime mechanism for Parsec to validate its clients. (We can then explain that future deployments will have more flexible means of protection, and talk about identity providers in that context). Without some context, I think there's a danger that these recommendations could seem arbitrary.

Reply:
Alright! I'll expand on the idea of Identity Provider

Reply:
I am personally more in favour of the option where we leave the Identity Provider sections but expand on what it is and the context. To do so, we can point out to the threat model where we make the same distinction of a deployment with and a without an IP:

> Parsec could also be deployed in an environment where all clients trust each other and there is no Identity Provider present, and this would lead to some changes to the Threat Model. These changes will be written in italic.

And also point to the [System Architecture](https://parallaxsecond.github.io/parsec-book/parsec_service/system_architecture.html#participating-components-and-their-roles) where we explain what the Identity Provider is.

Having to create a `parsec-clients` group and adding each and every client that wants to use `parsec` in it is quite a burden for administrators so it is nice, I think, for them to see "the light at the end of the tunnel" when an IP would exist.